logicrunner/handle_executorresults.go

```diff
@@ -86,8 +86,8 @@ func (p *initializeExecutionState) Proceed(ctx context.Context) error {
 	// prepare Queue
 	if p.msg.Queue != nil {
 		for _, qe := range p.msg.Queue {
-			ctxToSent := qe.Parcel.Context(context.Background())
-			transcript := NewTranscript(ctxToSent, qe.Parcel, qe.Request)
+			ctxToSent := context.TODO() //FIXME: !!!!
```

Comment:
why not context.Background() as it was before?

Reply:
created a task, INS-2944